translations/ui-eholdings/en.json

```diff
@@ -183,6 +183,10 @@
     "proxy": "Proxy",
     "proxy.inherited": "Inherited - {proxy}",
 
+    "accessType": "Access status type",
```

Comment:
This translation is already present, please use existing translation or update previous one